csrankings-h.csv

```diff
@@ -58,6 +58,7 @@ Haibin Kan,Fudan University,http://www.cs.fudan.edu.cn/?page_id=14441,NOSCHOLARP
 Haibin Ling,Stony Brook University,https://www3.cs.stonybrook.edu/~hling,jWx3qtIAAAAJ
 Haibing Guan,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/~hbguan,NOSCHOLARPAGE
 Haibo Chen 0001,Shanghai Jiao Tong University,http://ipads.se.sjtu.edu.cn/~hbchen,qd9xSkYAAAAJ
+Haibo Hu 0001,The Hong Kong Polytechnic University,https://haibohu.org/,N3DjoZYAAAAJ&hl

```

Comment:
Scholar ID needs to have &hl removed; faculty member not apparently in CS and not obviously eligible for inclusion.

https://www.polyu.edu.hk/comp/people/academic-staff/

Reply:
I have removed the '&hl'. Although Prof. Hu and Dr. Ye are in the Department of Electrical and Electronic Engineering of PolyU, they focus on database and security research. They frequently publish papers in top computer science conferences listed in CSRankings, such as VLDB, ICDE, S&P, SIGMOD, and NeurIPS. 

https://www.polyu.edu.hk/eee/people/academic-staff-and-teaching-staff/

Reply:
Thank you for your comment and we noted the inclusion criteria. Could you kindly assist by updating only the affiliation change for Prof. Man Ho Au (also known as Man Ho Allen Au) from The University of Hong Kong (HKU) to The Hong Kong Polytechnic University (PolyU), and omit the other two amendments concerning Prof. Hu and Dr. Ye?